### ISSUE TITLE
Datadog unix socket support

### ISSUE THREAD

**Issue by benwilson512** [spandex-project/spandex_datadog#17]
Hey folks!

Does this project support using unix sockets for data dog? I've got a PR into statix https://github.com/lexmag/statix/pull/35 to enable this, and I'd be happy to recreate that here.

**Comment by zachdaniel** [comment]
We don't support it, but if you'd like to add it that would be great :)

**Comment by GregMefford** [comment]
Yeah that looks neat! 👍 It's pretty common (I think) for folks to run the DD Agent on the same host as their Elixir app and point the traces at `localhost`, so a Unix socket might be better for that use-case.
